README.md

```diff
@@ -1,2 +1,98 @@
-# publishing-tutorial-sample-repo
-A sample repo to be used in conjunction with the tutorial "Getting started publishing with Pantheon 2" 
+# Publishing tutorial sample repo
+A sample repo to be used in conjunction with the tutorial "Getting started publishing with Pantheon 2"
+
+* [Contributing to this repository](#contributing-to-this-repository)
+
+## Contributing to this repository
+
+### Filing a bug
+
+If you have any suggestions to improve or extend this repository, add a new issue in the repository.
+
+### Prerequisites
+
+* You must have an account on GitHub.
```

Comment:
Do we need to say "you must"? 
* Have a GitHub account.
* Have your SSH key added to your GitHub account.

Reply:
Fixed.